pkg/preference/preference.go

```diff
@@ -154,6 +161,8 @@ type OdoSettings struct {
 
 	// RegistryCacheTime how long odo should cache information from registry
 	RegistryCacheTime *int `yaml:"RegistryCacheTime,omitempty"`
+
+	EphemeralSourceVolume *bool `yaml:"ephemeral,omitempty"`
```

Comment:
I think this needs to be `yaml:"Ephemeral,omitempty"`. Capital `E` vs. `e`. Asking because this is how it shows up:
```sh
odo preference -h
Modifies odo specific configuration settings within the global preference file.

 
Available Global Parameters:
 BuildTimeout - BuildTimeout (in seconds) for waiting for a build of the git component to complete (Default: 300)
 Experimental - Set this value to true to expose features in development/experimental mode
 NamePrefix - Use this value to set a default name prefix (Default: current directory name)
 PushTarget - Set this value to 'kube' or 'docker' to tell odo where to push applications to. (Default: kube)
 PushTimeout - PushTimeout (in seconds) for waiting for a Pod to come up (Default: 240)
 RegistryCacheTime - For how long (in minutes) odo will cache information from Devfile registry (Default: 15)
 Timeout - Timeout (in seconds) for OpenShift server connection check (Default: 1)
 UpdateNotification - Flag to control if an update notification is shown or not (Default: true)
 ephemeral - If true odo will create a emptyDir volume to store source code(Default: false)
```
Everything except `ephemeral` is capitalized.